src/main/kotlin/no/nav/tilleggsstonader/sak/fagsak/domain/Fagsak.kt

```diff
@@ -12,6 +12,7 @@ import org.springframework.data.relational.core.mapping.Table
 data class Fagsaker(
     val barnetilsyn: Fagsak?,
     val læremidler: Fagsak?,
+    val boUtgifter: Fagsak?,
```

Comment:
Burde kanskje kalle den for kun `boutgifter`? Husk å endre i sak-frontend også :-) 
```suggestion
    val boutgifter: Fagsak?,
```

Reply:
Oppdatert.